.changeset/angry-plums-retire.md

```diff
@@ -0,0 +1,16 @@
+---
+"@comet/cms-api": minor
+---
+
+Remove availablePermissions from UserPermissionsModule
+
+The `Permission`-interface and the `availablePermissions`-setting can simply be removed from the application.
+There is a `CometPermission`-export available to avoid using strings for the library-defined permissions. This object
+can be extended in the application to provide all permissions centralized:
```

Comment:
_if_ we suggest a centralized list of permissions (in starter, docs, changelog, migration guide etc) I'd prefer the interface variant.

If we decide against the interface, I wouldn't use such a central list and just put the strings in the decorators.

(comet should still have this export to be used by the application to add additional resolvers to the existing library permissions)